eq-author/src/constants/validationMessages.js

```diff
@@ -95,9 +95,15 @@ export const rightSideErrors = {
     errorCode: "ERR_RIGHTSIDE_ALLOFF_OR_NOT_ALLOWED",
     message: "All of not allowed in rule with checkbox OR option",
   },
-  ERR_RIGHTSIDE_AND_OR_NOT_ALLOWED: {
-    errorCode: "ERR_RIGHTSIDE_AND_OR_NOT_ALLOWED",
-    message: "All of not allowed in expression with checkbox OR option",
+  ERR_RIGHTSIDE_MIXING_OR_STND_OPTIONS_IN_AND_RULE: {
+    errorCode: "ERR_RIGHTSIDE_MIXING_OR_STND_OPTIONS_IN_AND_RULE",
+    message:
+      "You can't match 'all of' the standard options with the 'or' checkbox option. Change the 'All of' to 'Any of' or remove all of the standard options or the 'or' checkbox option",
+  },
+  ERR_GROUP_MIXING_EXPRESSIONS_WITH_OR_STND_OPTIONS_IN_AND: {
+    errorCode: "ERR_GROUP_MIXING_EXPRESSIONS_WITH_OR_STND_OPTIONS_IN_AND",
+    message:
+      "You can't match 'all of' the standard options with the 'or' checkbox option. Change the 'All of' to 'Any of' or remove a condition",
```

Comment:
Do the first 'all of' need to be capitalised like the second ones?

Reply:
Yes, it does! Sorry.